lib/src/components/NxFilterInput/NxFilterInput.tsx

```diff
@@ -29,18 +30,19 @@ const NxFilterInput = forwardRef<HTMLDivElement, Props>(
 
       return (
         <div {...otherProps} className={classes} ref={ref}>
-          <span className="nx-filter-input__add-on"
-                onClick={value && onClear || undefined}>
-            <NxFontAwesomeIcon icon={icon} />
-          </span>
-          <input type="text"
-                 autoComplete="off"
-                 id={inputId || undefined}
-                 value={value}
-                 onChange={inputOnChange}
-                 placeholder={placeholder || undefined}
-                 className="nx-filter-text-input"
-                 disabled={disabled || undefined} />
+          <div className="nx-text-input__box">
+            <span className="nx-filter-input__add-on" onClick={value && onClear || undefined}>
+              <NxFontAwesomeIcon icon={icon} className="nx-icon--filter-icons" />
```

Comment:
For the clear icon do we want to use our custom Close icon instead of FA?

Reply:
> For the clear icon do we want to use our custom Close icon instead of FA?

@dturner32 ?

Reply:
> > For the clear icon do we want to use our custom Close icon instead of FA?
> 
> @dturner32 ?

Note that it is not a "close' button, but a "clear" icon.